src/gstools/variogram/estimator.pyx

```diff
@@ -251,7 +250,7 @@ def unstructured(
     else:
         distance = dist_haversine
         if dim != 2:
-            raise ValueError('Haversine: dim = {0} != 2'.format(dim))
+            raise ValueError('Haversine: dim = {} != 2'.format(dim))
```

Comment:
Doesn't Cython understand f-strings as well? [link](http://blog.behnel.de/posts/whats-new-in-cython-024.html)

Reply:
I've manually corrected these, since yes, Cython has good f-string support